src/io/flutter/sdk/FlutterCommand.java

```diff
@@ -73,13 +73,17 @@ public String getDisplayCommand() {
    * <p>
    * If unable to start (for example, if a command is already running), returns null.
    */
-  public Process start(@Nullable Consumer<ProcessOutput> onDone) {
+  public Process start(@Nullable Consumer<ProcessOutput> onDone, @Nullable ProcessListener processListener) {
```

Comment:
Seems fine for now. If this gets any more complicated, it might be a bit more elegant to use a builder pattern to give optional parameters to a FlutterCommand. Something like:

sdk.flutterCreate(baseDir).withProcessListener(listener).whenDone(callback).start()